go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -77,6 +72,11 @@ func main() {
 			Name:  "init-gateways",
 			Usage: "initialize a gateway in the minion. Only useful with \"init-node\"",
 		},
+		cli.StringFlag{
```

Comment:
In which case would this be used? When a node deletion was missed? Can you document that?

Reply:
@squeed This is just moving the existing 'remove-node' code around. But I think you're right, we don't actually need this option anymore. The original commit was:

```
commit add0f897385c21b9dfee061ed31305a1d2ce11c7
Author:     Gurucharan Shetty <guru@ovn.org>
AuthorDate: Tue May 8 01:48:33 2018 -0700

    ovnkube: Ability to remove node.
    
    You can run this on a master node:
    ovnkube -remove-node=k8sminion2
    
    Or run this on any other node:
    ovnkube -remove-node=k8sminion2 \
    -nb-address=$NB_ADDRESS \
    -nb-client-privkey /etc/openvswitch/ovncontroller-privkey.pem \
    -nb-client-cert /etc/openvswitch/ovncontroller-cert.pem \
    -nb-client-cacert /etc/openvswitch/ovnnb-ca.cert
    
    The code does not try to remove the OVS bridge on the node. The
    thought process is that:
    1. We already have a 'ovn-kube-util bridges-to-nic' option for that.
    2. We want to be able to run this command on the master.
    3. We can in the future have a node watcher in master that will
    call this whenever a node gets deleted.
```

Reply:
If you power off a node, ovnkube master isn't running, and you remove the node from the Kube API, then you'll have some left-overs in the OVN database. This can be fixed with real DB reconciliation when the master comes back up, but we don't have that quite yet. However, we want to get that ASAP so I think we can probably kill this?

Reply:
sounds reasonable.